code/style.css

```diff
@@ -1,4 +1,320 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  color: rgb(8, 6, 6);  
+    }
+  
+* {
+  box-sizing: border-box;
+  }
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 auto;
+} 
+
+/**** Styling desktop ****/ 
+
+/**** Styling header ****/ 
+.header-tablet {
+  display: none;
 }
+
+.header {
+  
+   position: sticky;
+   top: 0;
+   border-bottom: 10px solid rgb(66, 65, 65);
+   background: rgb(121, 119, 119);
+}
+
+.sticky-top {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0px;
+  z-index: 99999;
+}
+
+.logo {
+  width: 100px;
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  width: 100%;
+}
+
+.menu-links {
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  list-style-type: none;
+}
+
+.link:hover {
+  background:blue;
+}
+
+.link {
+  padding: 20px;  
+}
+
+.link a {
+  color: white;
+  text-decoration: none;
+}
+
+.site-name {
+  margin-right: 350px; 
+  color:blue;
+}
+
+h1 {
+  font-size: 50px;
+ }
+
+p {
+  font-size: 22px;
+}
+
+h2 {
+  font-size: 25px;
+}
+
+/**** Styling big-news section ****/ 
+
+.big-news {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background: #f1f1f1;
+  margin: 35px;
+  padding: 15px 50px;
+}
+
+.big-news-container {
+  width: 50%;
+}
+
+.hide-on-desktop {
+  display: none;
+}
+
+.tesla-car {
+  width: 540px;
+}
+
+.big-news-text {
+  text-align: left;
+  padding-bottom: 25px; 
+  padding-left: 30px;
+ }
+
+/**** Styling news-cards ****/ 
+
+.other-news {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+}
+
+.news {
+  display: flex;
+  width: 24%;
+  padding: 20px 15px; 
+}
+
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2); 
+  background-color: #f1f1f1;
+  padding: 20px 20px 60px 20px;
+}
+
+.card:hover {
+ background-color: white;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.card img {
+  width: 100%; 
+  padding-bottom: 15px;  
+}
+
+/***Styling fotter-animation***/
+
+.animation {
+  overflow: hidden; 
+}
+
+h3 {
+  animation-duration: 3s;
+  animation-name: slidein;
+  color:  #f5900c;
+  text-align: center; 
+  padding: 20px;
+}
+
+@keyframes slidein {
+  from {
+    margin-left: 100%;
+    width: 300%;
+  }
+  to {
+    margin-left: 0%;
+    width: 100%;
+  }
+}
+
+/***Styling back-to-top button***/
+
+#btn {  
+  position: fixed;
+  bottom: 10px;
+  float: right;
+  right: 15.5%;
+  left: 77.25%;
+  max-width: 100px;
+  width: 100%;
+  font-size: 15px;
+  border-color: black;
+  background-color: #f09116;
+  padding: .5px;
+  border-radius: 4px;
+}
```

Comment:
Try the position: `right: 0;` and remove the `left: 77,25%` to not make it go out of the view:

<img width="396" alt="Skärmavbild 2021-02-03 kl  15 22 17" src="https://user-images.githubusercontent.com/51529201/106760049-aefb4480-6633-11eb-9da7-9b8e6cf80f36.png">